diff -ur mongoose.orig/Makefile mongoose/Makefile
--- mongoose.orig/Makefile	2009-03-16 16:56:39.000000000 -0600
+++ mongoose/Makefile	2009-03-16 16:59:25.000000000 -0600
@@ -20,7 +20,9 @@
 		$(SRCS) -ldl -lpthread -o $(PROG)
 
 bsd:
-	$(CC) $(COPT) $(CFLAGS) $(SRCS) -lpthread -o $(PROG)
+	$(CC) -c $(COPT) $(CFLAGS) mongoose.c 
+	$(AR) -r lib$(PROG).a *.o && ranlib lib$(PROG).a 
+	$(CC) $(COPT) $(CFLAGS) -L. -l$(PROG) -lpthread main.c -o $(PROG)
 
 rtems:
 	$(CC) -c $(COPT) $(CFLAGS) mongoose.c compat_rtems.c
Only in mongoose: Makefile~
Only in mongoose: libmongoose.a
Binary files mongoose.orig/mongoose and mongoose/mongoose differ
Only in mongoose: mongoose.o
